testlib/TesterMain.f90

```diff
@@ -221,6 +222,38 @@ subroutine assert_equal_str(estr,istr,test_description)
     endif
   end subroutine
 
+  !------------------------------------------------------------------
```

Comment:
I'm not sure changes to the `TesterMain.f90` within an exercise's directory will work in the test runner. @pclausen ?

Reply:
This file isn't in the exercise's directory.

Sorry for bundling the two things together.
I originally opened a separate pull request for the change to the testlib [#242](https://github.com/exercism/fortran/pull/242), but since that commit is in this pull request as well, I thought it might be unnecessary to recreate the other PR.

Reply:
No it's needed due to the fact that the `TesterMain.f90` is downloaded into the test runner I believe. @pclausen ?